packages/overlays/src/GlobalOverlayController.js

```diff
@@ -69,6 +73,8 @@ export class GlobalOverlayController extends BaseOverlayController {
       return;
     }
     if (!this.content.isConnected) {
+      this.content.classList.add('global-overlays__overlay-container');
```

Comment:
`global-overlays__overlay-container` is hardcoded twice, so let's abstract it to a variable?

Reply:
Where else do you see it?

Reply:
Where else do you see it?